src/features/import/utils/problems/predictProblems.spec.ts

```diff
@@ -459,6 +485,65 @@ describe('predictProblem()', () => {
     ]);
   });
 
+  it('returns problem when either the first name or last name value is missing', () => {
+    const sheet = makeFullSheet({
+      columns: [
+        {
+          field: 'first_name',
+          kind: ColumnKind.FIELD,
+          selected: true,
+        },
+        {
+          field: 'last_name',
+          kind: ColumnKind.FIELD,
+          selected: true,
+        },
+        {
+          field: 'email',
+          kind: ColumnKind.FIELD,
+          selected: true,
+        },
+      ],
+      rows: [
+        { data: ['Clara', 'Zetkin', 'zetkin@example.com'] },
+        { data: ['John', '', 'john@example.com'] },
+      ],
+    });
+
+    const problems = predictProblems(sheet, 'SE', []);
+    expect(problems).toEqual([
+      {
+        indices: [1],
+        kind: ImportProblemKind.MISSING_ID_AND_NAME,
+      },
+    ]);
+  });
+
+  it('returns correct problem when column is not configured', () => {
+    const sheet = makeFullSheet({
+      columns: [
+        {
+          kind: ColumnKind.UNKNOWN,
+          selected: false,
+        },
```

Comment:
The columns here does not match the data. The data contains three columns, but there is only one column here. There should be three columns here.

If this test is intended to accommodate [my comment from my previous review](https://github.com/zetkin/app.zetkin.org/pull/2117/files#r1776347820), there seems to have been a misunderstanding.

This test should contain three columns, the same three columns that the previous test, but with the first one (the one where first name is) de-selected.

```suggestion
        {
          kind: ColumnKind.UNKNOWN,
          selected: false,
        },
        {
          field: 'last_name',
          kind: ColumnKind.FIELD,
          selected: true,
        },
        {
          field: 'email',
          kind: ColumnKind.FIELD,
          selected: true,
        },
```